.github/workflows/ci.yaml

```diff
@@ -33,6 +37,8 @@ jobs:
       AFTER_SETUP_DOWNSTREAM_WORKSPACE: vcs pull $BASEDIR/downstream_ws/src
       # Clear the ccache stats before and log the stats after the build
       BEFORE_BUILD_UPSTREAM_WORKSPACE: ccache -z
+      BEFORE_INSTALL_UPSTREAM_DEPENDENCIES: export ROSDISTRO_INDEX_URL=https://raw.githubusercontent.com/ros/rosdistro/rolling/2022-01-28/index-v4.yaml
```

Comment:
@vatanaksoytezer I think you need to add this before the setup_rosdep job https://github.com/ros-planning/moveit2/runs/5202810205?check_suite_focus=true#step:12:137